ts_scripts/install_dependencies.py

```diff
@@ -128,12 +128,7 @@ def install_java(self):
             out = get_brew_version()
             if out == "N/A":
                 sys.exit("**Error: Homebrew not installed...")
-
-            os.system("brew tap AdoptOpenJDK/openjdk")
-            if out >= "2.7":
-                os.system("brew install --cask adoptopenjdk11")
-            else:
-                os.system("brew cask install adoptopenjdk11")
+            os.system("brew install openjdk@17")
```

Comment:
Why switch from adoptopenjdk to openjdk?

Reply:
Explained it in the description.